diff --git a/node_modules/react-native-gesture-handler/src/RNGestureHandlerModule.ts b/node_modules/react-native-gesture-handler/src/RNGestureHandlerModule.ts
index 83c6bbb..e5a0e7c 100644
--- a/node_modules/react-native-gesture-handler/src/RNGestureHandlerModule.ts
+++ b/node_modules/react-native-gesture-handler/src/RNGestureHandlerModule.ts
@@ -1,6 +1,6 @@
 import { NativeModules } from 'react-native';
 
-const RNGestureHandlerModule = NativeModules.RNGestureHandlerModule;
+const RNGestureHandlerModule = NativeModules.RNGestureHandlerModule || { install: () => {} };
 
 export default RNGestureHandlerModule;
